tests/integration/it_utils.py

```diff
@@ -60,6 +65,20 @@
 LEDGER_ACCEPT_REQUEST = GenericRequest(method="ledger_accept")
 
 
+class AsyncTestTimer:
+    def __init__(self, timeout, client):
+        self._timeout = timeout
+        self._client = client
+        self._task = asyncio.ensure_future(self._job())
+
+    async def _job(self):
+        await asyncio.sleep(self._timeout)
+        await self._client.request(LEDGER_ACCEPT_REQUEST)
+
+    def cancel(self):
```

Comment:
Is the task supposed to be canceled at some point?

Reply:
From my understanding, the task automatically cancels whenever the task completes. Should I keep the cancel function if I don't intend on using it? Unsure if it'll be useful in the future.

Reply:
That's fair, you can leave it in then